### PR TITLE
added the metakgp banner

### PR DESCRIPTION
**Pull request by aditimaurya**
@grapheo12 Can you please consider the latest commit for review. I have added the meta kgp banner on the footer along with the link of this GitHub project and metkgp's home page. But the link of the home page isn't working so kindly check it out once. And do let me know the edits are working properly or not, so I will do the further changes accordingly. Thanks!

### CODE REVIEW

File: templates/home.html [hunk 1]
```diff
@@ -142,6 +153,7 @@ <h4 class="unlockFont">Upload ICS file of your timetable from <a href="https://g
 					<input id="file-in" type="file" class="form-control col-auto" onchange="readICS()">
 				</div>
 				<div id="details-div"></div>
+				<p id="footer"> Contribute to this project on  Maintained by MetaKGP with ♥ </strong></p>
```
Comment (grapheo12) [inline templates/home.html:156]: I don't see any `<a>` tags here.

File: templates/home.html [hunk 2]
```diff
@@ -133,7 +133,18 @@ <h1>What Slot</h1>
...
+				  position: fixed; 
+				  padding: 13px 13px 0px 13px; 
+				  bottom: 0; 
+				  width: 100%; 
+												text-align: center;
+				  /* Height of the footer*/  
+				  height: 40px; 
+				  background: #ffffff; 
+				} 
+			  </style>
```
Comment (grapheo12) [inline templates/home.html:147]: It is very tempting to write inline CSS. But trust me, it becomes very hard to find them afterwards.
Better keep the CSS in `static/main.css`.

**Review by grapheo12 — CHANGES_REQUESTED** [review]

**Comment by grapheo12** [comment]
@aditimaurya It seems that you might have accidentally removed the URL itself.

**Comment by grapheo12** [comment]
@aditimaurya This silently also resolves #1 isn't it? It is a good practice to describe every minute detail of what you did in the PR, in the opening comment.

Check [this](https://github.com/grapheo12/iqps/pull/20) to better understand what I mean.

File: templates/home.html (hunk 2 above)
Comment (aditimaurya) [reply templates/home.html:147]: okay!!!!

**Comment by grapheo12** [comment]
I guess this can be closed since you have raised #6 .

**Comment by thealphadollar** [comment]
Closing this in favor of #6

**Pull request closed without merging**